src/windshaft/error.js

```diff
@@ -14,7 +14,7 @@ var WindshaftError = function (error) {
   }
   if (this.isAnalysisError(error.type)) {
     this.context = error.analysis && error.analysis.context;
-    this.analysisId = error.analysis && error.analysis.node_id;
+    this.analysisId = error.analysis && (error.analysis.node_id || error.analysis.id);
```

Comment:
Thanks for the clarification @dgaubert!

@ivanmalagon IMHO, we should not rely on `error.analysis.id` then. Instead, I agree with @dgaubert. Let's try to find an specific case where `error.analysis.node_id` is not being set correctly and fix it elsewhere.